Modules/_testclinic.c

```diff
@@ -1243,7 +1243,7 @@ _testclinic_TestClass_get_defining_class_arg_impl(PyObject *self,
                                                   PyObject *arg)
 /*[clinic end generated code: output=fe7e49d96cbb7718 input=d1b83d3b853af6d9]*/
 {
-    return Py_BuildValue("(OO)", cls, arg);
```

Comment:
Please remove this edit.  It is in deprecated code that will be removed soon.  So an optimization is pointless.